enumerable.rb

```diff
@@ -0,0 +1,24 @@
+require('./module')
```

Comment:
- [ ] It will be nice to use `require_relative` here. `require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory. 

Your outcome should be: `require_relative module`